diff -ur curl-7.63.0.orig/lib/url.c curl-7.63.0/lib/url.c
--- curl-7.63.0.orig/lib/url.c	2019-01-21 10:15:51.368019445 +0100
+++ curl-7.63.0/lib/url.c	2019-01-21 10:19:16.307523984 +0100
@@ -3057,6 +3057,7 @@
   }
 
   conn->bits.netrc = FALSE;
+#ifndef __BIONIC__
   if(data->set.use_netrc != CURL_NETRC_IGNORED &&
       (!*userp || !**userp || !*passwdp || !**passwdp)) {
     bool netrc_user_changed = FALSE;
@@ -3090,6 +3091,7 @@
       }
     }
   }
+#endif
 
   /* for updated strings, we update them in the URL */
   if(user_changed) {
